### PR TITLE
Start Authentication Testing with Jest and Supertest

### PR DESCRIPTION
**Pull request by Kader1680**
What I did
- Created initial tests for the authentication system.
- Covered:
  - User registration `/api/signup`
  - User login `/api/signin`
  - Admin role check `/api/isadmin`
 Tools Used
- Jest for unit testing.
- Supertest for HTTP requests.
- Used Mongoose to manually insert test users when needed.
 Notes
- This is the initial test setup; we will later add failure cases and edge case coverage.
- No real database data is affected — temporary test data is used.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -71,7 +71,9 @@ app.use("/api/order", orderRouter);
 app.use("/api/customize", customizeRouter);
 
 // Run Server
-const PORT = process.env.PORT || 8000;
-app.listen(PORT, () => {
-  console.log("Server is running on ", PORT);
-});
+// const PORT = process.env.PORT || 8000;
```
Comment (hasan-py) [inline server/app.js:74]: When the server will start now?? As you commented this?

**Review by hasan-py — CHANGES_REQUESTED** [review]
Thanks for adding the test. but could you follow up the request changed?

File: server/app.js (hunk 1 above)
Comment (Kader1680) [reply server/app.js:74]: yes i did the last code commented i can remove if you want

Comment (hasan-py) [reply server/app.js:74]: If we comment that our app will not run. Did you run all the code you changed? I was curious is that tested on your side.

**Comment by hasan-py** [comment]
As there has been no response after 3 weeks, I’m closing the PR. If you’ve fixed it, please reopen it.
